compiler/src/dotty/tools/dotc/interactive/Interactive.scala

```diff
@@ -131,7 +131,7 @@ object Interactive {
    *  @param includeReferences  If true, include references and not just definitions
    */
   def namedTrees(trees: List[SourceTree], includeReferences: Boolean, treePredicate: NameTree => Boolean)
-    (implicit ctx: Context): List[SourceTree] = {
+    (implicit ctx: Context): List[SourceTree] = safely {
```

Comment:
I think it's pretty rare that we would get a TypeError, so not a big deal either way.